appium/tests/screenobjects/new-message.screen.ts

```diff
@@ -277,7 +277,12 @@ class NewMessageScreen extends BaseScreen {
   }
 
   deleteAddedRecipientWithBackspace = async (order: number, type = 'to') => {
-    await this.showRecipientPopup(order, type)
+    await this.showRecipientPopup(order, type);
+    await driver.sendKeys(['\b']); // backspace
+  }
+
+  deleteAddedRecipientUsingBackSpaces = async () => {
+    await driver.sendKeys(['\b']); // backspace
     await driver.sendKeys(['\b']); // backspace
   }
```

Comment:
The names of these two methods should be distinguished further. It's not clear what the difference is.

Reply:
I suggest `deleteLastAddedRecipient` or ``deleteLastAddedRecipientUsingBackspace`

Reply:
The original could be called `deleteAddedRecipientByOpeningPopup` ? Else why is the popup there? But in that case, if using a popup, why use backspace, why not tap on delete icon?

Reply:
Are we testing delete with `open popup` -> `tap delete icon` ? That would be more important to test than `open popup` -> `backspace`

Reply:
Yes.
`Are we testing delete with open popup -> tap delete icon`

Reply:
Actually test both.

Reply:
Ok. That means we have three of these methods? They should all be called distinctly.

- `deleteAddedRecipientWithDoubleBackspace`
- `deleteAddedRecipientByTappingDeleteIconInModal`
- `deleteAddedRecipientWithBackspaceInModal`

Reply:
Yes